### PR TITLE
[docs][yba] Update alert configuration

### PR DESCRIPTION
**Pull request by ddhodge**
Update Alert configuration, including customizing templates and custom variables

@netlify /preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md [hunk 1]
```diff
@@ -11,447 +11,183 @@ menu:
...
-- [Alerts](../../alerts-monitoring/alert/)
-- [Metrics](../../troubleshoot/universe-issues/#use-metrics/)
-- [Alerts and Notifications in YugabyteDB Anywhere](https://www.yugabyte.com/blog/yugabytedb-2-8-alerts-and-notifications/)
-
-You can use preconfigured alerts provided by YugabyteDB Anywhere, or create and configure your own alerts based on the metrics' conditions.
-
-You can access YugabyteDB Anywhere health monitor and configure alerts by navigating to **Admin > Alert Configurations**, as per the following illustration:
-
-![Configure alerts](/images/yp/config-alerts1.png)
+You can access YugabyteDB Anywhere health monitor and configure alerts by navigating to **Admin > Alert Configurations**.
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md:18]: To me, health monitor is confusing because there is a separate health tab on **Admin > Alert Configurations > Health**.  So when you actually get to Admin > Alert Configurations , You see "Alert policies" as the default selection and maybe you can rephrase it to "You can access YugabyteDB Anywhere access policies and configure alerts by navigating to **Admin > Alert Configurations** "

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+menu:
+  preview_yugabyte-platform:
+    identifier: alert-policy-templates
+    parent: set-up-alerts-health-checking
+    weight: 40
+type: docs
+---
+
+Alert policies use templates to define how the alert is triggered. The alert templates have been created using Prometheus expressions.
+
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:15]: Will be useful to mention where you can see these policy templates from the UI or even link to [create-alerts](https://deploy-preview-17128--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check/#create-alerts) to find this list of templates.

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+Client to node certificate for universe `'$universe_name'` expires in `$value` days.
+
+```expression
+min by (node_name) (ybp_health_check_c2n_cert_validity_days{universe_uuid="$uuid"} < 30)
+```
+
+### Health check notification error
+
+Failed to issue health check notification for universe `'$universe_name'`. You need to check Health notification settings and YugabyteDB Anywhere logs for details or contact the Yugabyte support team.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:58]: link to Yugabyte Support team

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+
+```expression
+last_over_time(ybp_health_check_notification_status{universe_uuid = "$uuid"}[1d]) < 1
+```
+
+### Health check error
+
+Failed to perform health check for universe `'$universe_name'`. You need to check YugabyteDB Anywhere logs for details or contact the Yugabyte support team.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:66]: link to Yugabyte Support team

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+
+Core files detected for universe `'$universe_name'` on `$value` T-Server instances.
+
+```expression
+ybp_health_check_tserver_core_files{universe_uuid="$uuid"} > 0
+```
+
+### Alert notification failed
+
+Last attempt to send alert notifications for customer `'yugabyte support'` failed. You need to check YugabyteDB Anywhere logs for details or contact the Yugabyte support team.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:114]: link to Yugabyte Support team

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 6]
```diff
@@ -0,0 +1,306 @@
...
+
+Database queues overflow has been detected for universe `'$universe_name'`.
+
+```expression
+sum by (node_prefix) (increase(rpcs_queue_overflow{node_prefix="$node_prefix"}[10m])) + sum by (node_prefix) (increase(rpcs_timed_out_in_queue{node_prefix="$node_prefix"}[10m])) > 1
+```
+
+### Alert rules sync failed
+
+Last alert rules synchronization for customer `'yugabyte support'` has failed. YugabyteDB Anywhere logs for details or contact the Yugabyte support team.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:162]: link to Yugabyte Support team

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md [hunk 7]
```diff
@@ -0,0 +1,306 @@
...
+
+```expression
+last_over_time(ybp_alert_config_writer_status[1d]) < 1
+```
+
+### Alert query failed
+
+Last alert query for customer `'yugabyte support'` failed. YugabyteDB Anywhere logs for details or contact the Yugabyte support team.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md:170]: link to Yugabyte Support team

**Review by aishwarya24 — APPROVED** [review]
Approved with some comments.

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md [hunk 8]
```diff
@@ -11,447 +11,184 @@ menu:
 type: docs
 ---
 
-YugabyteDB Anywhere can check universes for issues that may affect deployment. Should problems arise, YugabyteDB Anywhere can automatically issue alert notifications
+YugabyteDB Anywhere has default, preconfigured alerts, both at the YugabyteDB Anywhere and universe level. Universe alerts can be configured globally for all universes, or per specific universe. In addition to the default alerts, you can configure your own alerts based on a specific condition on any available metric.
```
Comment (anmalysh-yb) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md:14]: on any available alert policy template. 

We can't configure alerts based on any metric right now.

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md [hunk 9]
```diff
@@ -11,447 +11,184 @@ menu:
...
+      - Enter a PagerDuty API key and service integration key.
 
-  If you select **Email** as a notification delivery method, perform the following:
+    If you select **WebHook** as a notification delivery method, perform the following:
 
-  - Provide a descriptive name for your channel.
+      - Provide a descriptive name for your channel.
 
-  - Use the **Emails** field to enter one or more valid email addresses separated by commas.
+      - Use the **Webhook URL** field to enter a valid URL.
```
Comment (anmalysh-yb) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md:120]: We added support for authentication config for webhook channel.
It looks like this:

image
image

May want to add this info here as well.

**Review by anmalysh-yb — APPROVED** [review]
Awesome!
